docs/networking.md

```diff
@@ -92,6 +93,26 @@ $ kubectl create -f https://git.io/weave-kube
 
 The above daemonset installation requires K8s 1.4.x or above.
 
+### Flannel Example for CNI and Network Policy
+
+#### Installing Flannel on a new Cluster
+
+The following command setups a cluster, in HA mode, with Flannel as the CNI and Network Policy provider.
+
+```console
+$ export $ZONES=mylistofzones
+$ kops create cluster \
+  --zones $ZONES \
+  --master-zones $ZONES \
+  --master-size m4.large \
+  --node-size m4.large \
+  --networking flannel \
+  --yes \
+  --name myclustername.mydns.io
+```
+
+The above will deploy a daemonset installation which requires K8s 1.4.x or above.
```

Comment:
Still confused ... we can talk offline.  Not confused about the PR, confused about what is actually in the CNI distro